### PR TITLE
bug: correct gallery title

### PR DESCRIPTION
**Pull request by M-Waweru**
- [x]  Things are good to go

> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

File: gallery-page.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Galley Page</title>
```
Comment (mutumakeffa) [inline gallery-page.html:7]: what is this?

File: gallery-page.html [hunk 2]
```diff
@@ -4,9 +4,9 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Galley Page</title>
+    <title>Gallery Page</title>
 </head>
 <body>
-    <h1>Galley Page</h1>
+    <h1>Gallery Page</h1>
```
Comment (mutumakeffa) [inline gallery-page.html:10]: Looks good now

File: gallery-page.html [hunk 3]
```diff
@@ -4,9 +4,9 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (mutumakeffa) [inline gallery-page.html:6]:
```suggestion
    <meta name="" content="width=device-width, initial-scale=1.0">

```

File: gallery-page.html (hunk 1 above)
Comment (M-Waweru) [reply gallery-page.html:7]: Just a typo

File: gallery-page.html (hunk 2 above)
Comment (M-Waweru) [reply gallery-page.html:10]: Thanks

File: gallery-page.html (hunk 1 above)
Comment (mutumakeffa) [reply gallery-page.html:7]: sasawa

**Pull request merged**